tasks/pocket_sand.py

```diff
@@ -0,0 +1,33 @@
+import os
+import random
+from discord_webhook import DiscordWebhook, DiscordEmbed
+import asyncio
+from tasks.base import BaseTask
+
+
+class DaleAttack(BaseTask):
```

Comment:
Not sure I like the class name being different than the package/file name unless there's going to be multiple different pocket_sand tasks